ci-operator/step-registry/storage/create/aws-extra-disks/storage-create-aws-extra-disks-commands.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+export AWS_SHARED_CREDENTIALS_FILE="${CLUSTER_PROFILE_DIR}/.awscred"
+
+# For disconnected or otherwise unreachable environments, we want to
+# have steps use an HTTP(S) proxy to reach the API server. This proxy
+# configuration file should export HTTP_PROXY, HTTPS_PROXY, and NO_PROXY
+# environment variables, as well as their lowercase equivalents (note
+# that libcurl doesn't recognize the uppercase variables).
+if test -f "${SHARED_DIR}/proxy-conf.sh"
+then
+	# shellcheck disable=SC1090
+	source "${SHARED_DIR}/proxy-conf.sh"
+fi
+
+# Create an AWS EBS volume, volInfraTag will help delete volume when deleting cluster 
+function create_ebs_volume(){
+  if [ $# -ne 6 ] ;then
+    echo "Usage: create_ebs_volume <volName> <availabilityZone> <volSize> <volType> <volInfraTag>" && return 1
```

Comment:
`$#` check for 6 params, while the example show 5.  Is it expected ?

Reply:
Good catch, yeah it should be 6 params, missed the `region`! Fixed, thanks!